r3bbase/R3BTprevTnext.cxx

```diff
@@ -0,0 +1,170 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2023 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+#include "R3BTprevTnext.h"
+#include "FairLogger.h"
+#include "FairRootManager.h"
+#include "FairRunAna.h"
+#include "FairRuntimeDb.h"
+#include "R3BEventHeader.h"
+#include "R3BMSOffsetPar.h"
+#include "R3BSamplerMappedData.h"
+#include "R3BShared.h"
+#include "TClonesArray.h"
+#include "TMath.h"
+#include "TObjArray.h"
+#include "TRandom.h"
+#include <cstdlib>
+#include <iostream>
+
+ClassImp(R3BTprevTnext) // NOLINT
+
+    R3BTprevTnext::R3BTprevTnext()
+    : R3BTprevTnext("R3B Tprev/Tnext", 1)
+{
+}
+
+R3BTprevTnext::R3BTprevTnext(const TString& name, Int_t iVerbose)
+    : FairTask(name, iVerbose)
+    , fMSOffsetPar(nullptr)
+    , fTprev(0.0)
+    , fTnext(0.0)
+    , fSamplerMapped(nullptr)
+    , fSamplerMSMapped(nullptr)
+    , fR3BEventHeader(nullptr)
+    , fTpat(1024.0)
+    , delta_clk(1.0)
+{
+}
```

Comment:
While I agree with @YanzhaoW in principle, I would also like to point out that this is among the bad practices used in the ./template/ directory. I think it would be worthwhile to clean that up, then we are in a much better position to display rightful indignation if PRs still use that :-)

Reply:
Is the inline initialization here only changing the comma position to the end of the lines instead of the beginning?

Reply:
You could see the correct way [here](https://en.cppreference.com/w/cpp/language/data_members#Member_initialization) (second one is preferred). It's very simple. You just need to define the default values inside the class definition. ;D